manifests/o/OpenChemistry/Avogadro2/1.99.0/OpenChemistry.Avogadro2.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: OpenChemistry.Avogadro2
+PackageVersion: 1.99.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-02-11
+Installers:
+- Architecture: x64
+  InstallerUrl: http://matebook-x-pro/d/programs/Avogadro2-1.99.0-win64.exe?sign=jOYBLNXHXjifcOaG13gPqQma6lo9ZFgKIUCMmDsdOmI=:0
```

```suggestion
  InstallerUrl: https://github.com/OpenChemistry/avogadroapp/releases/download/1.99.0/Avogadro2-1.99.0-win64.exe
```

Forget to switch back the official URL 🤣